header.css

```diff
@@ -0,0 +1,51 @@
+
+.linia {
+	height: 400px;
+	background-image: url(tlo1_small.jpg);
+	border: 2px solid black;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+@media screen and ( min-width: 1024px){
+.linia {
+	background-image: url(tlo1_big.jpg);
+	height: 550px;
+
+}
+}
+.linia1, .linia2, .linia3 {
```

Comment:
Selektor całkiem niezły, ale zastanówmy się czy jesteśmy w stanie zrobić to lepiej. Wszystkie trzy elementy to bezpośrednie i jedyne dzieci `h1` w naszym głównym headerze. Skorzystałbym tutaj z bardzo fajnej właściwości jaką oferuje CSS czyli dziedziczenia. W dziedziczeniu oczywiście chodzi o to, że elementy podrzędne mogą przybrać właściwości swego rodzica. Oczywiście nie wszystkie właściwości, ale `font-size` i `letter-spacing` zdecydowanie należą do właściwości dziedziczonych.
A oto ich pełna lista: https://gist.github.com/dcneiner/1137601
W takim przypadku możemy skrócić ten selektor i usunąć niepotrzebne klasy w HTML.

Jeszcze innym rozwiązaniem jest też stworzenie generycznej klasy tak jak w przypadku `color1` czy `text2` i dodanie jej tym trzem elementom. W każdym razie zawsze powinniśmy dążyć i stosować rozwiązania, które tworzą jak najmniej kodu CSS i HTML.